contrib/internal/httptrace/config_test.go

```diff
@@ -0,0 +1,92 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package httptrace
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfig(t *testing.T) {
+	restore := cleanEnv()
+	defaultCfg := newConfig()
+	restore()
```

Comment:
Instead of bothering with this, why not declare the default config clearly on the spot? 
```go
defaultCfg := &config{
    // ...
}
```
It's a small easy to write structure.

Reply:
Right. Doing so, I think it makes sense to keep the defaultQueryStringRegexp global so that it can be re-used here.